### PR TITLE
Fix docker-compose

### PR DESCRIPTION
**Pull request by paulopaquielli**
Fixed bug when application runs with docker-compose. And I added the alpine version in DockerFile to improve app start time.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-FROM node:carbon
+FROM node:carbon-alpine
```
Comment (rafaelverger) [inline Dockerfile]: I don't know if this will implicate somehow on how app runs on production env 🤔

Comment (RodolfoSilva) [reply Dockerfile]: Some packages need gcc, g++ and other things for compile the binary code. This will not work fine with carbon-alpine.

Comment (rafaelverger) [reply Dockerfile]: @PauloPaquielli remove those changes about docker image... stick to the changes related with issue #23

Comment (paulopaquielli) [reply Dockerfile]: Done @rafaelverger @RodolfoSilva. Sorry for the delayed response :)

**Comment by rafaelverger** [comment]
LGTM for me... @di3goleite , @RodolfoSilva ?

**Comment by di3goleite** [comment]
LGTM 🎉

**Pull request merged**